src/components/Layouts/MainLayout/MatchingFeed/MatchingFeed.tsx

```diff
@@ -218,64 +125,81 @@ const MatchingFeed = () => {
+        activeButton={activeButton}
+        setActiveButton={setActiveButton}
+        setSelectedProfileId={setSelectedProfileId}
+        switchTab={switchTab}
+        profilesData={profilesData}
+        isShowPremiumModal={isShowPremiumModal}
+      />
+      <ModalSubscription
+        isOpen={isShowPremiumModal}
+        isThereACancel={true}
+        onChange={setIsShowPremiumModal}
+        textTitle="Become a Premium"
+        textSubtitle="To view people who are «Interested in you» you need to subscribe to a Premium account"
+        buttonText="Learn more"
+        darkModal={true}
+        fcFirstBtn={handleChangeLearnMoreModal}
+        onClose={closePremiumModal}
+      >
+        <img src={logo} alt={logo} className={styles.logo} />
+      </ModalSubscription>
+      <ModalBox
+        open={isShowLearnMoreModal}
+        handleChangeOpen={handleChangeLearnMoreModal}
+        className={{
+          maxWidth: '432px',
+          backgroundColor: 'white',
+        }}
+      >
+        <div className={styles.learnMoreModalContainer}>
+          <div className={styles.learnMoreModalHeader}>
+            <span className={styles.learnMoreModalTitle}>
+              Fill out your account
+            </span>
+            <CloseIcon
+              onClick={handleChangeLearnMoreModal}
+              style={{ cursor: 'pointer' }}
+            />
           </div>
-          <div
-            className={`${styles.navCategory} ${
-              activeButton === 'NewPeople' ? styles.newPeopleActive : ''
-            }`}
-            onClick={() => switchTab('NewPeople')}
+          <p className={styles.learnMoreModalText}>
+            Before your profile becomes active and you can use all the features
+            of our app, you need to add at least one video.
+          </p>
+          <Button
+            className={styles.button}
+            onClick={handleChangeLearnMoreModal}
           >
-            New People
-          </div>
-        </div>
-        <div className={styles.matchingProfiles}>
-          {profilesData.map((profile) => (
-            <MatchedProfile
-              key={profile.id}
-              profile={profile}
-              setSelectedProfileId={setSelectedProfileId}
-              onDelete={() => handleDeleteProfile()}
-            />
-          ))}
+            Add video
+          </Button>
         </div>
-      </div>
-      <div>
-        <LikeButton
-          onClick={(event: React.MouseEvent<HTMLButtonElement>) => {
-            event.stopPropagation();
-            handleLikeProfile();
-          }}
-        />
-      </div>
+      </ModalBox>
       {showMatchModal && (
         <IsMatchedModal
-          closeMatchedModal={closeMatchedModal}
+          closeMatchedModal={handleCloseMatchModal}
           selectedProfileId={selectedProfileId}
           profilesData={profilesData}
         />
       )}
+      <LikeMatchButton
+        onClick={(event: MouseEvent<HTMLButtonElement>) => {
```

Comment:
it should work with infer types
```suggestion
        onClick={(event) => {
```

Reply:
agreed